src/devtools/views/Profiler/ProfilerContext.js

```diff
@@ -115,18 +115,15 @@ function ProfilerContextController({ children }: Props) {
   if (importedProfilingData !== null) {
     rootHasProfilingData = true;
   } else if (selectedElementID) {
-    rendererID = store.getRendererIDForElement(
-      ((selectedElementID: any): number)
-    );
-    rootID = store.getRootIDForElement(((selectedElementID: any): number));
-    rootHasProfilingData = store.profilingOperations.has(
-      ((rootID: any): number)
-    );
+    rendererID = store.getRendererIDForElement(selectedElementID);
+    rootID = store.getRootIDForElement(selectedElementID);
```

Comment:
Seems like the null checks should be done here rather than in the above signatures.

Reply:
OK.